docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -22,7 +22,7 @@ your applications and libraries. In this tutorial, you will add new metrics and
 **Prerequisites**: [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet) or a later version
```

Comment:
Should we update '.NET Core 3.1' to '.NET 6'?  .NET Core 3.1 is no longer supported.